stories/components/Time.stories.tsx

```diff
@@ -10,6 +10,9 @@ import Time from '../../src/components/time'
 export default {
     title: 'Components/Time',
     component: Time,
+    parameters: {
+        badges: ['accessible'],
+    },
```

Comment:
Does this one have jest-axe tests already?

Reply:
Nope, added.